vars/goMake.groovy

```diff
@@ -10,6 +10,8 @@ def call(body) {
     def ghOrg =  config.githubOrganisation
     def dockerOrg = config.dockerOrganisation
     def prj = config.project
+    def makeCommand = config.makeCommand ?: 'make'
+    def installPreRequisites = config.installPreRequisites
 
```

Comment:
https://github.com/fabric8io/go-builder/pull/4 opened PR for adding jq on the image